ant/libmoai-clean.sh

```diff
@@ -3,4 +3,4 @@ set -e # exit on error
 rm -rf libmoai/obj
 rm -rf libmoai/libs
 
-../util/pito ant-libmoai
```

Comment:
must have been a bad merge. ch had temporarily set some things back to moaiutil so the build server could continue working. must have somehow landed on top of my change back. or I missed it...